pages/android/bugreportz.md

```diff
@@ -12,7 +12,7 @@
 
 `bugreportz -p`
 
-- Show the version of `bugreportz`:
+- Display version:
```

Comment:
I think the original wording was better here.

Reply:
@sbrl for help and version descriptions we prefer using this generic wording in most cases to be uniform with other pages (https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#help-and-version-commands). We still suggest verbose descriptions for descriptions like `Display help for a subcommand:`, etc.